src/Pages/hokyunkim/Main/Main.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom'
+import './Main.scss'
+
+class Main extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            commentList: [],
+            commentValue: "", 
+        };
+    }
+
+
+    handleCommentValue = (event) => {
+        this.setState(
+            {
+            commentValue: event.target.value,
+            
+        })
+    };
+
+    handleBtn = (event) => {
+        this.setState({
+            commentList: [
+                ...this.state.commentList, 
+                {
+                    userName: "리액트 너무 어려워 나는 포기할래:",
+                    content: this.state.commentValue 
+                }
+            ],
+            commentValue:"",
+        });
+    };
+
+    handleEnterValue = (event) => {
+        if(event.key === "Enter"){
+            this.handleBtn()
+        }
+    }
+
+    render() {
+    
+
+       return(
+        <div className= "mainComment">
+        <h1>Comment Page</h1>
+        <ul>
+            {this.state.commentList.map((comment) => {
+            return (
+            <li>
```

Comment:
map 사용할 떄는 항상 key 속성 부여해주셔야 해요!
```suggestion
            <li key={comment.id}>
```